### PR TITLE
AR-4004 - switched out the documentation to what we gave partner

### PR DESCRIPTION
**Pull request by JGruberNXT**
@JakeDriscoll 
@ericlizotte 
@jsmet 
@VinnieMaida 
@hemangshahnxt 
@BobCardillo

### CODE REVIEW

File: source/includes/_patientrecall.md [hunk 1]
```diff
@@ -4,516 +4,151 @@
...
-GET https://select.nextech-api.com/api/patient-recall
+GET https://select.nextech-api.com/api/patient-recall?subject=Patient/5AAE9E3C-B1E4-46EA-93C2-CF3B36747D1A
 </pre>
 &nbsp;
 
-#### Example: Get a specific patient recall based on identifier
+#### Example: Get all recalls between and including 1/1/2019 through 5/31/2019
 
 <pre class="center-column">
-GET https://select.nextech-api.com/api/patient-recall/55
```
Comment (JakeDriscoll) [inline source/includes/_patientrecall.md]: Shouldn't this remain documented?

File: source/includes/_patientrecall.md [hunk 2]
```diff
@@ -4,516 +4,151 @@
...
+		"extension" : [
+		{
+			"url" : "http://hl7.org/fhir/StructureDefinition/Practitioner",
+			"valueResourceReference": {
+				  "reference": "Practitioner/9978",
+				  "display": "Davidson, Darren"
+			}
+		},
+		{
+			"url" : "http://hl7.org/fhir/StructureDefinition/Location",
```
Comment (JakeDriscoll) [inline source/includes/_patientrecall.md]: You mixed tabs in with spaces, can we make it consistent?
image

**Review by JakeDriscoll — CHANGES_REQUESTED** [review]
Two quick things

File: source/includes/_patientrecall.md (hunk 2 above)
Comment (JGruberNXT) [reply source/includes/_patientrecall.md]: fixed

File: source/includes/_patientrecall.md (hunk 1 above)
Comment (JGruberNXT) [reply source/includes/_patientrecall.md]: fixed

**Review by JakeDriscoll — APPROVED** [review]

**Pull request merged**